### ISSUE TITLE
[FEATURE] - [Neutral Dreams]

### ISSUE THREAD

**Issue by statecouncil** [QarsoonMeel/DreamersDreams#27]
Since +0 thoughts don't appear (that correct?), they could give something like +/- 1 or 2.
Neutral dreams should pull from all random character bios, possibly weighted or limited by origin pending on the pawn that has the dream. I will look at the bio list more closely to see grammatical problems that cannot be solved simply by replacing the languageworker commands.

**Comment by QarsoonMeel** [comment]
I need to investigate whether these can be displayed or not.

Another issue is that I'm planning on translating this mod. Pulling from backstories can be risky in this case. English is easy to sort out but what about Polish? The grammar is so much complex.
